Changes.md

```diff
@@ -1,3 +1,15 @@
+1.x.x.x (relative to 1.1.6.1)
+=======
+
+Features
+--------
+
+- Cycles :
+  - Added use_camera and use_caustics for lights.
+  - Fixed distant light angle is in degrees and not radians.
```

Comment:
To be entirely pedantic, this should probably go under a `Fixes` header.

Reply:
Done